#0 building with "default" instance using docker driver

#1 [internal] load build definition from Dockerfile
#1 transferring dockerfile: 2.47kB done
#1 DONE 0.0s

#2 [internal] load metadata for docker.io/library/python:3.10
#2 DONE 0.0s

#3 [internal] load .dockerignore
#3 transferring context: 2B done
#3 DONE 0.0s

#4 [internal] load build context
#4 transferring context: 30.20kB done
#4 DONE 0.0s

#5 [ 1/12] FROM docker.io/library/python:3.10
#5 CACHED

#6 [ 2/12] COPY search_patch /search_patch
#6 DONE 0.0s

#7 [ 3/12] COPY code_edit.py /code_edit.py
#7 DONE 0.0s

#8 [ 4/12] RUN mkdir -p ~/.pip && touch ~/.pip/pip.conf &&     echo "[global]" >> ~/.pip/pip.conf && echo "index-url=http://simple/" >> ~/.pip/pip.conf && echo "[install]" >> ~/.pip/pip.conf && echo "trusted-host=pypi..org" >> ~/.pip/pip.conf &&     apt-get update && apt-get install -y curl &&     curl -sSL https://install.python-poetry.org | python -
#8 0.504 Get:1 http://deb.debian.org/debian bookworm InRelease [151 kB]
#8 0.858 Get:2 http://deb.debian.org/debian bookworm-updates InRelease [55.4 kB]
#8 1.012 Get:3 http://deb.debian.org/debian-security bookworm-security InRelease [48.0 kB]
#8 1.108 Get:4 http://deb.debian.org/debian bookworm/main amd64 Packages [8789 kB]
#8 1.629 Get:5 http://deb.debian.org/debian bookworm-updates/main amd64 Packages [8856 B]
#8 1.780 Get:6 http://deb.debian.org/debian-security bookworm-security/main amd64 Packages [240 kB]
#8 2.933 Fetched 9292 kB in 3s (3583 kB/s)
#8 2.933 Reading package lists...
#8 3.729 Reading package lists...
#8 4.528 Building dependency tree...
#8 4.768 Reading state information...
#8 5.145 curl is already the newest version (7.88.1-10+deb12u8).
#8 5.145 0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
#8 26.61 Retrieving Poetry metadata
#8 26.61 
#8 26.61 # Welcome to Poetry!
#8 26.61 
#8 26.61 This will download and install the latest version of Poetry,
#8 26.61 a dependency and package manager for Python.
#8 26.61 
#8 26.61 It will add the `poetry` command to Poetry's bin directory, located at:
#8 26.61 
#8 26.61 /root/.local/bin
#8 26.61 
#8 26.61 You can uninstall at any time by executing this script with the --uninstall option,
#8 26.61 and these changes will be reverted.
#8 26.61 
#8 26.61 Installing Poetry (2.0.0)
#8 26.61 Installing Poetry (2.0.0): Creating environment
#8 26.61 Installing Poetry (2.0.0): Installing Poetry
#8 26.61 Installing Poetry (2.0.0): Creating script
#8 26.61 Installing Poetry (2.0.0): Done
#8 26.61 
#8 26.61 Poetry (2.0.0) is installed now. Great!
#8 26.61 
#8 26.61 To get started you need Poetry's bin directory (/root/.local/bin) in your `PATH`
#8 26.61 environment variable.
#8 26.61 
#8 26.61 Add `export PATH="/root/.local/bin:$PATH"` to your shell configuration file.
#8 26.61 
#8 26.61 Alternatively, you can call Poetry explicitly with `/root/.local/bin/poetry`.
#8 26.61 
#8 26.61 You can test that everything is set up by executing:
#8 26.61 
#8 26.61 `poetry --version`
#8 26.61 
#8 DONE 27.0s

#9 [ 5/12] RUN pip install pytest pytest-xdist &&     pip install pipdeptree &&     git clone https://github.com/bytewiz3/TravelGPT.git &&     mkdir /repo &&     git config --global --add safe.directory /repo &&     cp -r /TravelGPT/. /repo && rm -rf /TravelGPT/ &&     rm -rf /TravelGPT &&     cd /repo && git checkout b19b43095fd0343d7a0d045f59ec869c8b1fda33 &&     pip install loguru==0.6.0 &&     pip install pydantic==1.10.15 &&     pip install PyJWT==2.8.0 &&     pip install PyMySQL==1.1.0 &&     pip install PyYAML==6.0.1 &&     pip install SQLAlchemy==1.4.52 &&     pip install fastapi==0.92.0 &&     pip install openai==1.16.2 &&     pip install starlette==0.25.0 &&     pip install requests==2.31.0 &&     pip install uvicorn==0.29.0 &&     pip install httpx==0.27.2 &&     pip install python-dotenv==1.0.1 &&     pip install Jinja2==3.1.4 &&     pip install sse-starlette==2.0.0 &&     pip install redis==5.0.8 &&     pip install pocode==0.0.3
#9 2.005 Looking in indexes: http://simple/
#9 2.159 Collecting pytest
#9 2.178   Downloading http://packages/pytest/pytest-8.3.4-py3-none-any.whl (343 kB)
#9 2.202      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 343.1/343.1 kB 17.1 MB/s eta 0:00:00
#9 2.274 Collecting pytest-xdist
#9 2.297   Downloading http://packages/pytest-xdist/pytest_xdist-3.6.1-py3-none-any.whl (46 kB)
#9 2.309      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 46.1/46.1 kB 4.6 MB/s eta 0:00:00
#9 2.388 Collecting packaging
#9 2.411   Downloading http://packages/packaging/packaging-24.2-py3-none-any.whl (65 kB)
#9 2.423      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 65.5/65.5 kB 6.8 MB/s eta 0:00:00
#9 2.450 Collecting iniconfig
#9 2.470   Downloading http://packages/iniconfig/iniconfig-2.0.0-py3-none-any.whl (5.9 kB)
#9 2.515 Collecting exceptiongroup>=1.0.0rc8
#9 2.538   Downloading http://packages/exceptiongroup/exceptiongroup-1.2.2-py3-none-any.whl (16 kB)
#9 2.598 Collecting tomli>=1
#9 2.616   Downloading http://packages/tomli/tomli-2.2.1-py3-none-any.whl (14 kB)
#9 2.664 Collecting pluggy<2,>=1.5
#9 2.683   Downloading http://packages/pluggy/pluggy-1.5.0-py3-none-any.whl (20 kB)
#9 2.746 Collecting execnet>=2.1
#9 2.790   Downloading http://packages/execnet/execnet-2.1.1-py3-none-any.whl (40 kB)
#9 2.798      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 40.6/40.6 kB 7.5 MB/s eta 0:00:00
#9 2.968 Installing collected packages: tomli, pluggy, packaging, iniconfig, execnet, exceptiongroup, pytest, pytest-xdist
#9 3.646 Successfully installed exceptiongroup-1.2.2 execnet-2.1.1 iniconfig-2.0.0 packaging-24.2 pluggy-1.5.0 pytest-8.3.4 pytest-xdist-3.6.1 tomli-2.2.1
#9 3.647 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
#9 3.782 
#9 3.782 [notice] A new release of pip is available: 23.0.1 -> 24.3.1
#9 3.782 [notice] To update, run: pip install --upgrade pip
#9 4.701 Looking in indexes: http://simple/
#9 4.781 Collecting pipdeptree
#9 4.800   Downloading http://packages/pipdeptree/pipdeptree-2.24.0-py3-none-any.whl (32 kB)
#9 4.824 Requirement already satisfied: packaging>=24.1 in /usr/local/lib/python3.10/site-packages (from pipdeptree) (24.2)
#9 4.939 Collecting pip>=24.2
#9 4.949   Using cached http://packages/pip/pip-24.3.1-py3-none-any.whl (1.8 MB)
#9 5.069 Installing collected packages: pip, pipdeptree
#9 5.069   Attempting uninstall: pip
#9 5.070     Found existing installation: pip 23.0.1
#9 5.228     Uninstalling pip-23.0.1:
#9 5.405       Successfully uninstalled pip-23.0.1
#9 6.879 Successfully installed pip-24.3.1 pipdeptree-2.24.0
#9 6.880 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
#9 7.089 Cloning into 'TravelGPT'...
#9 10.47 Note: switching to 'b19b43095fd0343d7a0d045f59ec869c8b1fda33'.
#9 10.47 
#9 10.47 You are in 'detached HEAD' state. You can look around, make experimental
#9 10.47 changes and commit them, and you can discard any commits you make in this
#9 10.47 state without impacting any branches by switching back to a branch.
#9 10.47 
#9 10.47 If you want to create a new branch to retain commits you create, you may
#9 10.47 do so (now or later) by using -c with the switch command. Example:
#9 10.47 
#9 10.47   git switch -c <new-branch-name>
#9 10.47 
#9 10.47 Or undo this operation with:
#9 10.47 
#9 10.47   git switch -
#9 10.47 
#9 10.47 Turn off this advice by setting config variable advice.detachedHead to false
#9 10.47 
#9 10.47 HEAD is now at b19b430 Merge pull request #96 from bytewiz3/feature-update
#9 11.03 Looking in indexes: http://simple/
#9 11.07 Collecting loguru==0.6.0
#9 11.09   Downloading http://packages/loguru/loguru-0.6.0-py3-none-any.whl (58 kB)
#9 11.14 Installing collected packages: loguru
#9 11.20 Successfully installed loguru-0.6.0
#9 11.20 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager, possibly rendering your system unusable.It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv. Use the --root-user-action option if you know what you are doing and want to suppress this warning.
#9 11.91 Looking in indexes: http://simple/
#9 12.24 Collecting pydantic==1.10.15
#9 12.26   Downloading http://packages/pydantic/pydantic-1.10.15-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (3.1 MB)
#9 12.32      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 3.1/3.1 MB 81.9 MB/s eta 0:00:00
#9 12.38 Collecting typing-extensions>=4.2.0 (from pydantic==1.10.15)
#9 12.40   Downloading http://packages/typing-extensions/typing_extensions-4.12.2-py3-none-any.whl (37 kB)
#9 12.46 Installing collected packages: typing-extensions, pydantic
#9 12.70 Successfully installed pydantic-1.10.15 typing-extensions-4.12.2
#9 12.70 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager, possibly rendering your system unusable.It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv. Use the --root-user-action option if you know what you are doing and want to suppress this warning.
#9 13.50 Looking in indexes: http://simple/
#9 13.55 Collecting PyJWT==2.8.0
#9 13.57   Downloading http://packages/pyjwt/PyJWT-2.8.0-py3-none-any.whl (22 kB)
#9 13.62 Installing collected packages: PyJWT
#9 13.65 Successfully installed PyJWT-2.8.0
#9 13.65 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager, possibly rendering your system unusable.It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv. Use the --root-user-action option if you know what you are doing and want to suppress this warning.
#9 14.38 Looking in indexes: http://simple/
#9 14.43 Collecting PyMySQL==1.1.0
#9 14.44   Downloading http://packages/pymysql/PyMySQL-1.1.0-py3-none-any.whl (44 kB)
#9 14.50 Installing collected packages: PyMySQL
#9 14.54 Successfully installed PyMySQL-1.1.0
#9 14.54 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager, possibly rendering your system unusable.It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv. Use the --root-user-action option if you know what you are doing and want to suppress this warning.
#9 15.22 Looking in indexes: http://simple/
#9 15.32 Collecting PyYAML==6.0.1
#9 15.34   Downloading http://packages/pyyaml/PyYAML-6.0.1-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (705 kB)
#9 15.36      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 705.5/705.5 kB 37.4 MB/s eta 0:00:00
#9 15.44 Installing collected packages: PyYAML
#9 15.52 Successfully installed PyYAML-6.0.1
#9 15.52 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager, possibly rendering your system unusable.It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv. Use the --root-user-action option if you know what you are doing and want to suppress this warning.
#9 16.43 Looking in indexes: http://simple/
#9 17.19 Collecting SQLAlchemy==1.4.52
#9 17.35   Downloading http://packages/sqlalchemy/SQLAlchemy-1.4.52-cp310-cp310-manylinux_2_5_x86_64.manylinux1_x86_64.manylinux_2_17_x86_64.manylinux2014_x86_64.whl (1.6 MB)
#9 17.39      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 1.6/1.6 MB 65.8 MB/s eta 0:00:00
#9 17.72 Collecting greenlet!=0.4.17 (from SQLAlchemy==1.4.52)
#9 17.75   Downloading http://packages/greenlet/greenlet-3.1.1-cp310-cp310-manylinux_2_24_x86_64.manylinux_2_28_x86_64.whl (599 kB)
#9 17.78      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 599.5/599.5 kB 26.2 MB/s eta 0:00:00
#9 17.87 Installing collected packages: greenlet, SQLAlchemy
#9 19.02 Successfully installed SQLAlchemy-1.4.52 greenlet-3.1.1
#9 19.02 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager, possibly rendering your system unusable.It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv. Use the --root-user-action option if you know what you are doing and want to suppress this warning.
#9 19.85 Looking in indexes: http://simple/
#9 19.99 Collecting fastapi==0.92.0
#9 20.07   Downloading http://packages/fastapi/fastapi-0.92.0-py3-none-any.whl (56 kB)
#9 20.13 Requirement already satisfied: pydantic!=1.7,!=1.7.1,!=1.7.2,!=1.7.3,!=1.8,!=1.8.1,<2.0.0,>=1.6.2 in /usr/local/lib/python3.10/site-packages (from fastapi==0.92.0) (1.10.15)
#9 20.20 Collecting starlette<0.26.0,>=0.25.0 (from fastapi==0.92.0)
#9 20.28   Downloading http://packages/starlette/starlette-0.25.0-py3-none-any.whl (66 kB)
#9 20.31 Requirement already satisfied: typing-extensions>=4.2.0 in /usr/local/lib/python3.10/site-packages (from pydantic!=1.7,!=1.7.1,!=1.7.2,!=1.7.3,!=1.8,!=1.8.1,<2.0.0,>=1.6.2->fastapi==0.92.0) (4.12.2)
#9 20.35 Collecting anyio<5,>=3.4.0 (from starlette<0.26.0,>=0.25.0->fastapi==0.92.0)
#9 20.37   Downloading http://packages/anyio/anyio-4.8.0-py3-none-any.whl (96 kB)
#9 20.40 Requirement already satisfied: exceptiongroup>=1.0.2 in /usr/local/lib/python3.10/site-packages (from anyio<5,>=3.4.0->starlette<0.26.0,>=0.25.0->fastapi==0.92.0) (1.2.2)
#9 20.43 Collecting idna>=2.8 (from anyio<5,>=3.4.0->starlette<0.26.0,>=0.25.0->fastapi==0.92.0)
#9 20.43   Using cached http://packages/idna/idna-3.10-py3-none-any.whl (70 kB)
#9 20.45 Collecting sniffio>=1.1 (from anyio<5,>=3.4.0->starlette<0.26.0,>=0.25.0->fastapi==0.92.0)
#9 20.47   Downloading http://packages/sniffio/sniffio-1.3.1-py3-none-any.whl (10 kB)
#9 20.58 Installing collected packages: sniffio, idna, anyio, starlette, fastapi
#9 21.01 Successfully installed anyio-4.8.0 fastapi-0.92.0 idna-3.10 sniffio-1.3.1 starlette-0.25.0
#9 21.01 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager, possibly rendering your system unusable.It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv. Use the --root-user-action option if you know what you are doing and want to suppress this warning.
#9 21.74 Looking in indexes: http://simple/
#9 21.88 Collecting openai==1.16.2
#9 21.97   Downloading http://packages/openai/openai-1.16.2-py3-none-any.whl (267 kB)
#9 22.03 Requirement already satisfied: anyio<5,>=3.5.0 in /usr/local/lib/python3.10/site-packages (from openai==1.16.2) (4.8.0)
#9 22.05 Collecting distro<2,>=1.7.0 (from openai==1.16.2)
#9 22.07   Downloading http://packages/distro/distro-1.9.0-py3-none-any.whl (20 kB)
#9 22.13 Collecting httpx<1,>=0.23.0 (from openai==1.16.2)
#9 22.16   Downloading http://packages/httpx/httpx-0.28.1-py3-none-any.whl (73 kB)
#9 22.18 Requirement already satisfied: pydantic<3,>=1.9.0 in /usr/local/lib/python3.10/site-packages (from openai==1.16.2) (1.10.15)
#9 22.18 Requirement already satisfied: sniffio in /usr/local/lib/python3.10/site-packages (from openai==1.16.2) (1.3.1)
#9 22.28 Collecting tqdm>4 (from openai==1.16.2)
#9 22.30   Downloading http://packages/tqdm/tqdm-4.67.1-py3-none-any.whl (78 kB)
#9 22.32 Requirement already satisfied: typing-extensions<5,>=4.7 in /usr/local/lib/python3.10/site-packages (from openai==1.16.2) (4.12.2)
#9 22.33 Requirement already satisfied: exceptiongroup>=1.0.2 in /usr/local/lib/python3.10/site-packages (from anyio<5,>=3.5.0->openai==1.16.2) (1.2.2)
#9 22.33 Requirement already satisfied: idna>=2.8 in /usr/local/lib/python3.10/site-packages (from anyio<5,>=3.5.0->openai==1.16.2) (3.10)
#9 22.37 Collecting certifi (from httpx<1,>=0.23.0->openai==1.16.2)
#9 22.38   Using cached http://packages/certifi/certifi-2024.12.14-py3-none-any.whl (164 kB)
#9 22.43 Collecting httpcore==1.* (from httpx<1,>=0.23.0->openai==1.16.2)
#9 22.45   Downloading http://packages/httpcore/httpcore-1.0.7-py3-none-any.whl (78 kB)
#9 22.49 Collecting h11<0.15,>=0.13 (from httpcore==1.*->httpx<1,>=0.23.0->openai==1.16.2)
#9 22.51   Downloading http://packages/h11/h11-0.14.0-py3-none-any.whl (58 kB)
#9 22.65 Installing collected packages: tqdm, h11, distro, certifi, httpcore, httpx, openai
#9 23.22 Successfully installed certifi-2024.12.14 distro-1.9.0 h11-0.14.0 httpcore-1.0.7 httpx-0.28.1 openai-1.16.2 tqdm-4.67.1
#9 23.22 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager, possibly rendering your system unusable.It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv. Use the --root-user-action option if you know what you are doing and want to suppress this warning.
#9 23.99 Looking in indexes: http://simple/
#9 23.99 Requirement already satisfied: starlette==0.25.0 in /usr/local/lib/python3.10/site-packages (0.25.0)
#9 24.00 Requirement already satisfied: anyio<5,>=3.4.0 in /usr/local/lib/python3.10/site-packages (from starlette==0.25.0) (4.8.0)
#9 24.01 Requirement already satisfied: exceptiongroup>=1.0.2 in /usr/local/lib/python3.10/site-packages (from anyio<5,>=3.4.0->starlette==0.25.0) (1.2.2)
#9 24.01 Requirement already satisfied: idna>=2.8 in /usr/local/lib/python3.10/site-packages (from anyio<5,>=3.4.0->starlette==0.25.0) (3.10)
#9 24.01 Requirement already satisfied: sniffio>=1.1 in /usr/local/lib/python3.10/site-packages (from anyio<5,>=3.4.0->starlette==0.25.0) (1.3.1)
#9 24.01 Requirement already satisfied: typing_extensions>=4.5 in /usr/local/lib/python3.10/site-packages (from anyio<5,>=3.4.0->starlette==0.25.0) (4.12.2)
#9 24.15 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager, possibly rendering your system unusable.It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv. Use the --root-user-action option if you know what you are doing and want to suppress this warning.
#9 24.88 Looking in indexes: http://simple/
#9 24.97 Collecting requests==2.31.0
#9 24.99   Downloading http://packages/requests/requests-2.31.0-py3-none-any.whl (62 kB)
#9 25.20 Collecting charset-normalizer<4,>=2 (from requests==2.31.0)
#9 25.20   Using cached http://packages/charset-normalizer/charset_normalizer-3.4.1-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (146 kB)
#9 25.21 Requirement already satisfied: idna<4,>=2.5 in /usr/local/lib/python3.10/site-packages (from requests==2.31.0) (3.10)
#9 25.30 Collecting urllib3<3,>=1.21.1 (from requests==2.31.0)
#9 25.30   Using cached http://packages/urllib3/urllib3-2.3.0-py3-none-any.whl (128 kB)
#9 25.31 Requirement already satisfied: certifi>=2017.4.17 in /usr/local/lib/python3.10/site-packages (from requests==2.31.0) (2024.12.14)
#9 25.43 Installing collected packages: urllib3, charset-normalizer, requests
#9 25.64 Successfully installed charset-normalizer-3.4.1 requests-2.31.0 urllib3-2.3.0
#9 25.64 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager, possibly rendering your system unusable.It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv. Use the --root-user-action option if you know what you are doing and want to suppress this warning.
#9 26.41 Looking in indexes: http://simple/
#9 26.48 Collecting uvicorn==0.29.0
#9 26.50   Downloading http://packages/uvicorn/uvicorn-0.29.0-py3-none-any.whl (60 kB)
#9 26.56 Collecting click>=7.0 (from uvicorn==0.29.0)
#9 26.58   Downloading http://packages/click/click-8.1.8-py3-none-any.whl (98 kB)
#9 26.59 Requirement already satisfied: h11>=0.8 in /usr/local/lib/python3.10/site-packages (from uvicorn==0.29.0) (0.14.0)
#9 26.59 Requirement already satisfied: typing-extensions>=4.0 in /usr/local/lib/python3.10/site-packages (from uvicorn==0.29.0) (4.12.2)
#9 26.75 Installing collected packages: click, uvicorn
#9 26.91 Successfully installed click-8.1.8 uvicorn-0.29.0
#9 26.91 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager, possibly rendering your system unusable.It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv. Use the --root-user-action option if you know what you are doing and want to suppress this warning.
#9 27.66 Looking in indexes: http://simple/
#9 27.72 Collecting httpx==0.27.2
#9 27.74   Downloading http://packages/httpx/httpx-0.27.2-py3-none-any.whl (76 kB)
#9 27.76 Requirement already satisfied: anyio in /usr/local/lib/python3.10/site-packages (from httpx==0.27.2) (4.8.0)
#9 27.76 Requirement already satisfied: certifi in /usr/local/lib/python3.10/site-packages (from httpx==0.27.2) (2024.12.14)
#9 27.76 Requirement already satisfied: httpcore==1.* in /usr/local/lib/python3.10/site-packages (from httpx==0.27.2) (1.0.7)
#9 27.76 Requirement already satisfied: idna in /usr/local/lib/python3.10/site-packages (from httpx==0.27.2) (3.10)
#9 27.76 Requirement already satisfied: sniffio in /usr/local/lib/python3.10/site-packages (from httpx==0.27.2) (1.3.1)
#9 27.77 Requirement already satisfied: h11<0.15,>=0.13 in /usr/local/lib/python3.10/site-packages (from httpcore==1.*->httpx==0.27.2) (0.14.0)
#9 27.77 Requirement already satisfied: exceptiongroup>=1.0.2 in /usr/local/lib/python3.10/site-packages (from anyio->httpx==0.27.2) (1.2.2)
#9 27.77 Requirement already satisfied: typing_extensions>=4.5 in /usr/local/lib/python3.10/site-packages (from anyio->httpx==0.27.2) (4.12.2)
#9 27.93 Installing collected packages: httpx
#9 27.93   Attempting uninstall: httpx
#9 27.93     Found existing installation: httpx 0.28.1
#9 27.93     Uninstalling httpx-0.28.1:
#9 27.94       Successfully uninstalled httpx-0.28.1
#9 28.03 Successfully installed httpx-0.27.2
#9 28.03 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager, possibly rendering your system unusable.It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv. Use the --root-user-action option if you know what you are doing and want to suppress this warning.
#9 28.72 Looking in indexes: http://simple/
#9 28.78 Collecting python-dotenv==1.0.1
#9 28.82   Downloading http://packages/python-dotenv/python_dotenv-1.0.1-py3-none-any.whl (19 kB)
#9 28.98 Installing collected packages: python-dotenv
#9 29.01 Successfully installed python-dotenv-1.0.1
#9 29.01 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager, possibly rendering your system unusable.It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv. Use the --root-user-action option if you know what you are doing and want to suppress this warning.
#9 29.76 Looking in indexes: http://simple/
#9 29.81 Collecting Jinja2==3.1.4
#9 29.83   Downloading http://packages/jinja2/jinja2-3.1.4-py3-none-any.whl (133 kB)
#9 29.98 Collecting MarkupSafe>=2.0 (from Jinja2==3.1.4)
#9 30.00   Downloading http://packages/markupsafe/MarkupSafe-3.0.2-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (20 kB)
#9 30.17 Installing collected packages: MarkupSafe, Jinja2
#9 30.32 Successfully installed Jinja2-3.1.4 MarkupSafe-3.0.2
#9 30.32 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager, possibly rendering your system unusable.It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv. Use the --root-user-action option if you know what you are doing and want to suppress this warning.
#9 31.09 Looking in indexes: http://simple/
#9 31.14 Collecting sse-starlette==2.0.0
#9 31.22   Downloading http://packages/sse-starlette/sse_starlette-2.0.0-py3-none-any.whl (9.0 kB)
#9 31.23 Requirement already satisfied: starlette in /usr/local/lib/python3.10/site-packages (from sse-starlette==2.0.0) (0.25.0)
#9 31.24 Requirement already satisfied: uvicorn in /usr/local/lib/python3.10/site-packages (from sse-starlette==2.0.0) (0.29.0)
#9 31.24 Requirement already satisfied: anyio in /usr/local/lib/python3.10/site-packages (from sse-starlette==2.0.0) (4.8.0)
#9 31.24 Requirement already satisfied: exceptiongroup>=1.0.2 in /usr/local/lib/python3.10/site-packages (from anyio->sse-starlette==2.0.0) (1.2.2)
#9 31.24 Requirement already satisfied: idna>=2.8 in /usr/local/lib/python3.10/site-packages (from anyio->sse-starlette==2.0.0) (3.10)
#9 31.25 Requirement already satisfied: sniffio>=1.1 in /usr/local/lib/python3.10/site-packages (from anyio->sse-starlette==2.0.0) (1.3.1)
#9 31.25 Requirement already satisfied: typing_extensions>=4.5 in /usr/local/lib/python3.10/site-packages (from anyio->sse-starlette==2.0.0) (4.12.2)
#9 31.25 Requirement already satisfied: click>=7.0 in /usr/local/lib/python3.10/site-packages (from uvicorn->sse-starlette==2.0.0) (8.1.8)
#9 31.25 Requirement already satisfied: h11>=0.8 in /usr/local/lib/python3.10/site-packages (from uvicorn->sse-starlette==2.0.0) (0.14.0)
#9 31.40 Installing collected packages: sse-starlette
#9 31.42 Successfully installed sse-starlette-2.0.0
#9 31.42 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager, possibly rendering your system unusable.It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv. Use the --root-user-action option if you know what you are doing and want to suppress this warning.
#9 32.11 Looking in indexes: http://simple/
#9 32.22 Collecting redis==5.0.8
#9 32.23   Downloading http://packages/redis/redis-5.0.8-py3-none-any.whl (255 kB)
#9 32.29 Collecting async-timeout>=4.0.3 (from redis==5.0.8)
#9 32.31   Downloading http://packages/async-timeout/async_timeout-5.0.1-py3-none-any.whl (6.2 kB)
#9 32.47 Installing collected packages: async-timeout, redis
#9 32.72 Successfully installed async-timeout-5.0.1 redis-5.0.8
#9 32.72 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager, possibly rendering your system unusable.It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv. Use the --root-user-action option if you know what you are doing and want to suppress this warning.
#9 33.43 Looking in indexes: http://simple/
#9 33.47 Collecting pocode==0.0.3
#9 33.62   Downloading http://packages/pocode/pocode-0.0.3-py3-none-any.whl (6.9 kB)
#9 33.67 Collecting colorama (from pocode==0.0.3)
#9 33.69   Downloading http://packages/colorama/colorama-0.4.6-py2.py3-none-any.whl (25 kB)
#9 33.85 Installing collected packages: colorama, pocode
#9 33.90 Successfully installed colorama-0.4.6 pocode-0.0.3
#9 33.90 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager, possibly rendering your system unusable.It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv. Use the --root-user-action option if you know what you are doing and want to suppress this warning.
#9 DONE 34.2s

#10 [ 6/12] RUN python /code_edit.py -t "/search_patch/0.patch"
#10 0.598 * Succeed Patch:
#10 0.598 Create and write /repo/common/utils/md5_utils.py!
#10 0.598 Create and write /repo/common/utils/valid_utils.py!
#10 0.598 
#10 DONE 0.7s

#11 [ 7/12] RUN python /code_edit.py -t "/search_patch/1.patch"
#11 0.502 * Succeed Patch:
#11 0.502 The new code snippet is inserted at the beginning of the file /repo/common/utils/email_utils.py
#11 0.502 --- repocommonutilsemail_utils.py
#11 0.502 +++ repocommonutilsemail_utils.py
#11 0.502 @@ -1,3 +1,6 @@
#11 0.502 +def send_email(subject, content, to_addrs):
#11 0.502 +    print(f"Mock send_email called with subject: {subject}, content: {content}, to: {to_addrs}")
#11 0.502 +    return True
#11 0.502  #!/usr/bin/python
#11 0.502  # -*- coding: UTF-8 -*-
#11 0.502  
#11 0.502 
#11 DONE 0.6s

#12 [ 8/12] RUN python /code_edit.py -t "/search_patch/2.patch"
#12 0.522 * Succeed Patch:
#12 0.522 --- arepocommonutilsemail_utils.py
#12 0.522 +++ brepocommonutilsemail_utils.py
#12 0.522 @@ -1,3 +1,4 @@
#12 0.522 +# Existing code to mock
#12 0.522  def send_email(subject, content, to_addrs):
#12 0.522      print(f"Mock send_email called with subject: {subject}, content: {content}, to: {to_addrs}")
#12 0.522      return True
#12 0.522 
#12 0.522 
#12 DONE 0.7s

#13 [ 9/12] RUN python /code_edit.py -t "/search_patch/3.patch" &&     python /code_edit.py -t "/search_patch/4.patch"
#13 0.520 * Succeed Patch:
#13 0.520 The new code snippet is inserted at the beginning of the file /repo/common/utils/email_utils.py
#13 0.520 --- repocommonutilsemail_utils.py
#13 0.520 +++ repocommonutilsemail_utils.py
#13 0.520 @@ -1,3 +1,6 @@
#13 0.520 +def send_email(subject, content, to_addrs):
#13 0.520 +    print(f"Mock send_email called with subject: {subject}, content: {content}, to: {to_addrs}")
#13 0.520 +    return True
#13 0.520  # Existing code to mock
#13 0.520  def send_email(subject, content, to_addrs):
#13 0.520      print(f"Mock send_email called with subject: {subject}, content: {content}, to: {to_addrs}")
#13 0.520 
#13 0.593 * Succeed Patch:
#13 0.593 The new code snippet is inserted at the beginning of the file /repo/common/utils/email_utils.py
#13 0.593 --- repocommonutilsemail_utils.py
#13 0.593 +++ repocommonutilsemail_utils.py
#13 0.593 @@ -1,3 +1,7 @@
#13 0.593 +# Mock send_email to avoid network call
#13 0.593 +def send_email(subject, content, to_addrs):
#13 0.593 +    print(f"Mock send_email called with subject: {subject}, content: {content}, to: {to_addrs}")
#13 0.593 +    return True
#13 0.593  def send_email(subject, content, to_addrs):
#13 0.593      print(f"Mock send_email called with subject: {subject}, content: {content}, to: {to_addrs}")
#13 0.593      return True
#13 0.593 
#13 DONE 0.7s

#14 [10/12] RUN python /code_edit.py -t "/search_patch/5.patch" &&     python /code_edit.py -t "/search_patch/6.patch" &&     python /code_edit.py -t "/search_patch/7.patch" &&     rm /repo/common/utils/email_utils.py &&     python /code_edit.py -t "/search_patch/8.patch"
#14 0.530 * Succeed Patch:
#14 0.530 * Fail Patch:
#14 0.530 #SEARCH/REPLACE block failed to match!
#14 0.530 
#14 0.530 ## SearchReplaceNoExactMatch: This SEARCH block failed to exactly match lines in /repo/common/utils/email_utils.py
#14 0.530 # Mock send_email to avoid network call
#14 0.530 def send_email(subject, content, to_addrs):
#14 0.530     print(f"Mock send_email called with subject: {subject}, content: {content}, to: {to_addrs}")
#14 0.530     return True
#14 0.530 
#14 0.530 # Existing code to mock
#14 0.530 def send_email(subject, content, to_addrs):
#14 0.530     print(f"Mock send_email called with subject: {subject}, content: {content}, to: {to_addrs}")
#14 0.530     return True
#14 0.530 
#14 0.530 
#14 0.530 Did you mean to match some of these actual lines from /repo/common/utils/email_utils.py?
#14 0.530 
#14 0.530 ```diff
#14 0.530 # Mock send_email to avoid network call
#14 0.530 def send_email(subject, content, to_addrs):
#14 0.530     print(f"Mock send_email called with subject: {subject}, content: {content}, to: {to_addrs}")
#14 0.530     return True
#14 0.530 def send_email(subject, content, to_addrs):
#14 0.530     print(f"Mock send_email called with subject: {subject}, content: {content}, to: {to_addrs}")
#14 0.530     return True
#14 0.530 # Existing code to mock
#14 0.530 def send_email(subject, content, to_addrs):
#14 0.530     print(f"Mock send_email called with subject: {subject}, content: {content}, to: {to_addrs}")
#14 0.530     return True
#14 0.530 #!/usr/bin/python
#14 0.530 # -*- coding: UTF-8 -*-
#14 0.530 
#14 0.530 ```
#14 0.530 
#14 0.530 Are you sure you need this SEARCH/REPLACE block?
#14 0.530 The REPLACE lines are already in /repo/common/utils/email_utils.py!
#14 0.530 
#14 0.530 The SEARCH section must exactly match an existing block of lines including all white space, comments, indentation, docstrings, etc
#14 0.530 
#14 0.530 TIPS:
#14 0.530 * All patches must be based on the original code. Make sure it and try to provide more sufficient and unique old line(s) from snippet to facilitate matching.
#14 0.530 * Make sure your patch has right indentation.
#14 0.530 * Make sure your file path is correct.
#14 0.602 * Succeed Patch:
#14 0.602 * Fail Patch:
#14 0.602 #SEARCH/REPLACE block failed to match!
#14 0.602 
#14 0.602 ## SearchReplaceNoExactMatch: This SEARCH block failed to exactly match lines in /repo/common/utils/email_utils.py
#14 0.602 # Mock send_email to avoid network call
#14 0.602 def send_email(subject, content, to_addrs):
#14 0.602     print(f"Mock send_email called with subject: {subject}, content: {content}, to: {to_addrs}")
#14 0.602     return True
#14 0.602 def send_email(subject, content, to_addrs):
#14 0.602     print(f"Mock send_email called with subject: {subject}, content: {content}, to: {to_addrs}")
#14 0.602     return True
#14 0.602 # Existing code to mock
#14 0.602 def send_email(subject, content, to_addrs):
#14 0.602     print(f"Mock send_email called with subject: {subject}, content: {content}, to: {to_addrs}")
#14 0.602     return True
#14 0.602 #!/usr/bin/python
#14 0.602 # -*- coding: UTF-8 -*-
#14 0.602 import smtplib
#14 0.602 from email.mime.text import MIMEText
#14 0.602 from email.header import Header
#14 0.602 # Third-party SMTP service
#14 0.602 mail_host = "smtp.XXX.com"  # Set the server
#14 0.602 mail_user = "XXXX"  # Username
#14 0.602 mail_pass = "XXXXXX"  # Password
#14 0.602 sender = 'from@runoob.com'
#14 0.602 receivers = ['429240967@qq.com']  # Receive email, you can set it to your QQ email or other email
#14 0.602 message = MIMEText('Python email sending test...', 'plain', 'utf-8')
#14 0.602 message['From'] = Header("Tutorial", 'utf-8')
#14 0.602 message['To'] = Header("Test", 'utf-8')
#14 0.602 subject = 'Python SMTP email test'
#14 0.602 message['Subject'] = Header(subject, 'utf-8')
#14 0.602 try:
#14 0.602     smtpObj = smtplib.SMTP()
#14 0.602     smtpObj.connect(mail_host, 25)  # 25 is the SMTP port number
#14 0.602     smtpObj.login(mail_user, mail_pass)
#14 0.602     smtpObj.sendmail(sender, receivers, message.as_string())
#14 0.602     print
#14 0.602     "Email sent successfully"
#14 0.602 except smtplib.SMTPException:
#14 0.602     print
#14 0.602     "Error: Unable to send email"
#14 0.602 
#14 0.602 
#14 0.602 Did you mean to match some of these actual lines from /repo/common/utils/email_utils.py?
#14 0.602 
#14 0.602 ```diff
#14 0.602 # Mock send_email to avoid network call
#14 0.602 def send_email(subject, content, to_addrs):
#14 0.602     print(f"Mock send_email called with subject: {subject}, content: {content}, to: {to_addrs}")
#14 0.602     return True
#14 0.602 def send_email(subject, content, to_addrs):
#14 0.602     print(f"Mock send_email called with subject: {subject}, content: {content}, to: {to_addrs}")
#14 0.602     return True
#14 0.602 # Existing code to mock
#14 0.602 def send_email(subject, content, to_addrs):
#14 0.602     print(f"Mock send_email called with subject: {subject}, content: {content}, to: {to_addrs}")
#14 0.602     return True
#14 0.602 #!/usr/bin/python
#14 0.602 # -*- coding: UTF-8 -*-
#14 0.602 
#14 0.602 import smtplib
#14 0.602 from email.mime.text import MIMEText
#14 0.602 from email.header import Header
#14 0.602 
#14 0.602 # Third-party SMTP service
#14 0.602 mail_host = "smtp.XXX.com"  # Set the server
#14 0.602 mail_user = "XXXX"  # Username
#14 0.602 mail_pass = "XXXXXX"  # Password
#14 0.602 
#14 0.602 sender = 'from@runoob.com'
#14 0.602 receivers = ['429240967@qq.com']  # Receive email, you can set it to your QQ email or other email
#14 0.602 
#14 0.602 message = MIMEText('Python email sending test...', 'plain', 'utf-8')
#14 0.602 message['From'] = Header("Tutorial", 'utf-8')
#14 0.602 message['To'] = Header("Test", 'utf-8')
#14 0.602 
#14 0.602 subject = 'Python SMTP email test'
#14 0.602 message['Subject'] = Header(subject, 'utf-8')
#14 0.602 
#14 0.602 try:
#14 0.602     smtpObj = smtplib.SMTP()
#14 0.602     smtpObj.connect(mail_host, 25)  # 25 is the SMTP port number
#14 0.602     smtpObj.login(mail_user, mail_pass)
#14 0.602     smtpObj.sendmail(sender, receivers, message.as_string())
#14 0.602     print
#14 0.602     "Email sent successfully"
#14 0.602 except smtplib.SMTPException:
#14 0.602     print
#14 0.602 ```
#14 0.602 
#14 0.602 Are you sure you need this SEARCH/REPLACE block?
#14 0.602 The REPLACE lines are already in /repo/common/utils/email_utils.py!
#14 0.602 
#14 0.602 The SEARCH section must exactly match an existing block of lines including all white space, comments, indentation, docstrings, etc
#14 0.602 
#14 0.602 TIPS:
#14 0.602 * All patches must be based on the original code. Make sure it and try to provide more sufficient and unique old line(s) from snippet to facilitate matching.
#14 0.602 * Make sure your patch has right indentation.
#14 0.602 * Make sure your file path is correct.
#14 0.673 * Succeed Patch:
#14 0.673 * Fail Patch:
#14 0.673 #SEARCH/REPLACE block failed to match!
#14 0.673 
#14 0.673 ## SearchReplaceNoExactMatch: This SEARCH block failed to exactly match lines in /repo/common/utils/email_utils.py
#14 0.673 # Mock send_email to avoid network call
#14 0.673 def send_email(subject, content, to_addrs):
#14 0.673     print(f"Mock send_email called with subject: {subject}, content: {content}, to: {to_addrs}")
#14 0.673     return True
#14 0.673 def send_email(subject, content, to_addrs):
#14 0.673     print(f"Mock send_email called with subject: {subject}, content: {content}, to: {to_addrs}")
#14 0.673     return True
#14 0.673 # Existing code to mock
#14 0.673 def send_email(subject, content, to_addrs):
#14 0.673     print(f"Mock send_email called with subject: {subject}, content: {content}, to: {to_addrs}")
#14 0.673     return True
#14 0.673 #!/usr/bin/python
#14 0.673 # -*- coding: UTF-8 -*-
#14 0.673 import smtplib
#14 0.673 from email.mime.text import MIMEText
#14 0.673 from email.header import Header
#14 0.673 # Third-party SMTP service
#14 0.673 mail_host = "smtp.XXX.com"  # Set the server
#14 0.673 mail_user = "XXXX"  # Username
#14 0.673 mail_pass = "XXXXXX"  # Password
#14 0.673 sender = 'from@runoob.com'
#14 0.673 receivers = ['429240967@qq.com']  # Receive email, you can set it to your QQ email or other email
#14 0.673 message = MIMEText('Python email sending test...', 'plain', 'utf-8')
#14 0.673 message['From'] = Header("Tutorial", 'utf-8')
#14 0.673 message['To'] = Header("Test", 'utf-8')
#14 0.673 subject = 'Python SMTP email test'
#14 0.673 message['Subject'] = Header(subject, 'utf-8')
#14 0.673 try:
#14 0.673     smtpObj = smtplib.SMTP()
#14 0.673     smtpObj.connect(mail_host, 25)  # 25 is the SMTP port number
#14 0.673     smtpObj.login(mail_user, mail_pass)
#14 0.673     smtpObj.sendmail(sender, receivers, message.as_string())
#14 0.673     print
#14 0.673     "Email sent successfully"
#14 0.673 except smtplib.SMTPException:
#14 0.673     print
#14 0.673     "Error: Unable to send email"
#14 0.673 
#14 0.673 
#14 0.673 Did you mean to match some of these actual lines from /repo/common/utils/email_utils.py?
#14 0.673 
#14 0.673 ```diff
#14 0.673 # Mock send_email to avoid network call
#14 0.673 def send_email(subject, content, to_addrs):
#14 0.673     print(f"Mock send_email called with subject: {subject}, content: {content}, to: {to_addrs}")
#14 0.673     return True
#14 0.673 def send_email(subject, content, to_addrs):
#14 0.673     print(f"Mock send_email called with subject: {subject}, content: {content}, to: {to_addrs}")
#14 0.673     return True
#14 0.673 # Existing code to mock
#14 0.673 def send_email(subject, content, to_addrs):
#14 0.673     print(f"Mock send_email called with subject: {subject}, content: {content}, to: {to_addrs}")
#14 0.673     return True
#14 0.673 #!/usr/bin/python
#14 0.673 # -*- coding: UTF-8 -*-
#14 0.673 
#14 0.673 import smtplib
#14 0.673 from email.mime.text import MIMEText
#14 0.673 from email.header import Header
#14 0.673 
#14 0.673 # Third-party SMTP service
#14 0.673 mail_host = "smtp.XXX.com"  # Set the server
#14 0.673 mail_user = "XXXX"  # Username
#14 0.673 mail_pass = "XXXXXX"  # Password
#14 0.673 
#14 0.673 sender = 'from@runoob.com'
#14 0.673 receivers = ['429240967@qq.com']  # Receive email, you can set it to your QQ email or other email
#14 0.673 
#14 0.673 message = MIMEText('Python email sending test...', 'plain', 'utf-8')
#14 0.673 message['From'] = Header("Tutorial", 'utf-8')
#14 0.673 message['To'] = Header("Test", 'utf-8')
#14 0.673 
#14 0.673 subject = 'Python SMTP email test'
#14 0.673 message['Subject'] = Header(subject, 'utf-8')
#14 0.673 
#14 0.673 try:
#14 0.673     smtpObj = smtplib.SMTP()
#14 0.673     smtpObj.connect(mail_host, 25)  # 25 is the SMTP port number
#14 0.673     smtpObj.login(mail_user, mail_pass)
#14 0.673     smtpObj.sendmail(sender, receivers, message.as_string())
#14 0.673     print
#14 0.673     "Email sent successfully"
#14 0.673 except smtplib.SMTPException:
#14 0.673     print
#14 0.673 ```
#14 0.673 
#14 0.673 Are you sure you need this SEARCH/REPLACE block?
#14 0.673 The REPLACE lines are already in /repo/common/utils/email_utils.py!
#14 0.673 
#14 0.673 The SEARCH section must exactly match an existing block of lines including all white space, comments, indentation, docstrings, etc
#14 0.673 
#14 0.673 TIPS:
#14 0.673 * All patches must be based on the original code. Make sure it and try to provide more sufficient and unique old line(s) from snippet to facilitate matching.
#14 0.673 * Make sure your patch has right indentation.
#14 0.673 * Make sure your file path is correct.
#14 0.744 * Succeed Patch:
#14 0.744 Create and write /repo/common/utils/email_utils.py!
#14 0.744 
#14 DONE 0.9s

#15 [11/12] RUN cd /repo && pytest --collect-only -q
#15 1.181 tests/test_chat_list.py::TestChatPromptListEndpoint::test_get_prompt_list_invalid_method
#15 1.181 tests/test_chat_list.py::TestChatPromptListEndpoint::test_get_prompt_list_unauthorized
#15 1.181 tests/test_chat_list.py::TestChatPromptListEndpoint::test_get_prompt_list_valid_request
#15 1.181 tests/test_chat_list.py::TestChatPromptListEndpoint::test_update_prompt_empty_new_prompt
#15 1.181 tests/test_chat_list.py::TestChatPromptListEndpoint::test_update_prompt_html_content
#15 1.181 tests/test_chat_list.py::TestChatPromptListEndpoint::test_update_prompt_internal_server_error
#15 1.181 tests/test_chat_list.py::TestChatPromptListEndpoint::test_update_prompt_javascript_content
#15 1.181 tests/test_chat_list.py::TestChatPromptListEndpoint::test_update_prompt_json_content
#15 1.181 tests/test_chat_list.py::TestChatPromptListEndpoint::test_update_prompt_long_new_prompt
#15 1.181 tests/test_chat_list.py::TestChatPromptListEndpoint::test_update_prompt_minimum_length_prompt
#15 1.181 tests/test_chat_list.py::TestChatPromptListEndpoint::test_update_prompt_nonexistent_id
#15 1.181 tests/test_chat_list.py::TestChatPromptListEndpoint::test_update_prompt_null_new_prompt
#15 1.181 tests/test_chat_list.py::TestChatPromptListEndpoint::test_update_prompt_special_characters
#15 1.181 tests/test_chat_list.py::TestChatPromptListEndpoint::test_update_prompt_sql_injection
#15 1.181 tests/test_chat_list.py::TestChatPromptListEndpoint::test_update_prompt_unauthorized
#15 1.181 tests/test_chat_list.py::TestChatPromptListEndpoint::test_update_prompt_whitespace_content
#15 1.181 tests/test_circum_address_list.py::TestUserRegister::test_address_empty_vals
#15 1.181 tests/test_circum_address_list.py::TestUserRegister::test_address_no_params
#15 1.181 tests/test_circum_address_list.py::TestUserRegister::test_circum_address_sucess1
#15 1.181 tests/test_circum_address_list.py::TestUserRegister::test_circum_address_sucess2
#15 1.181 tests/test_delete_chat_history.py::TestChatHistoryEndpoint::test_retrieve_chat_history_invalid_method
#15 1.181 tests/test_delete_chat_history.py::TestChatHistoryEndpoint::test_retrieve_chat_history_invalid_user_id
#15 1.181 tests/test_delete_chat_history.py::TestChatHistoryEndpoint::test_retrieve_chat_history_missing_parameters
#15 1.181 tests/test_delete_chat_history.py::TestChatHistoryEndpoint::test_retrieve_chat_history_no_history
#15 1.181 tests/test_delete_chat_history.py::TestChatHistoryEndpoint::test_retrieve_chat_history_unauthorized
#15 1.181 tests/test_delete_chat_history.py::TestChatHistoryEndpoint::test_retrieve_chat_history_valid_request
#15 1.181 tests/test_email_utils.py::EmailUtilsTest::test_send_email
#15 1.181 tests/test_get_chat_history.py::TestChatHistoryEndpoint::test_retrieve_chat_history_invalid_method
#15 1.181 tests/test_get_chat_history.py::TestChatHistoryEndpoint::test_retrieve_chat_history_invalid_user_id
#15 1.181 tests/test_get_chat_history.py::TestChatHistoryEndpoint::test_retrieve_chat_history_missing_parameters
#15 1.181 tests/test_get_chat_history.py::TestChatHistoryEndpoint::test_retrieve_chat_history_no_history
#15 1.181 tests/test_get_chat_history.py::TestChatHistoryEndpoint::test_retrieve_chat_history_unauthorized
#15 1.181 tests/test_get_chat_history.py::TestChatHistoryEndpoint::test_retrieve_chat_history_valid_request
#15 1.181 tests/test_md5_utils.py::Md5UtilsTest::test_1
#15 1.181 tests/test_md5_utils.py::Md5UtilsTest::test_2
#15 1.181 tests/test_md5_utils.py::Md5UtilsTest::test_email3
#15 1.181 tests/test_save_chat_history.py::TestSaveChatHistory::test_save_chat_history_invalid_data
#15 1.181 tests/test_save_chat_history.py::TestSaveChatHistory::test_save_chat_history_missing_fields
#15 1.181 tests/test_save_chat_history.py::TestSaveChatHistory::test_save_chat_history_success
#15 1.181 tests/test_user_login.py::TestUserRegister::test_user_login_invalid_password
#15 1.181 tests/test_user_login.py::TestUserRegister::test_user_login_invalid_username
#15 1.181 tests/test_user_login.py::TestUserRegister::test_user_login_missing_password
#15 1.181 tests/test_user_login.py::TestUserRegister::test_user_login_missing_username
#15 1.181 tests/test_user_login.py::TestUserRegister::test_user_login_success
#15 1.181 tests/test_user_register.py::TestUserRegister::test_register_duplicate_user
#15 1.181 tests/test_user_register.py::TestUserRegister::test_register_invalid_password1
#15 1.181 tests/test_user_register.py::TestUserRegister::test_register_invalid_password2
#15 1.181 tests/test_user_register.py::TestUserRegister::test_register_invalid_password3
#15 1.181 tests/test_user_register.py::TestUserRegister::test_register_missing_username
#15 1.181 tests/test_user_register.py::TestUserRegister::test_register_success
#15 1.181 tests/test_valid_utils.py::EmailUtilsTest::test_email1
#15 1.181 tests/test_valid_utils.py::EmailUtilsTest::test_email2
#15 1.181 tests/test_valid_utils.py::EmailUtilsTest::test_email3
#15 1.181 
#15 1.181 53 tests collected in 0.17s
#15 DONE 1.4s

#16 exporting to image
#16 exporting layers
#16 exporting layers 1.0s done
#16 writing image sha256:59a870cedc0ba7f451351d9569074676667cfacb37bcc1ae578011b8e9217083 done
#16 naming to docker.io/library/tmp:tmp done
#16 DONE 1.1s
